src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -3055,20 +3053,13 @@ public enum RisingRivals implements LogicCardInfo {
               before KNOCKOUT, {
```

Comment:
Water Rescue has some rulings that you should double check work properly and don't cause engine errors:
https://compendium.pokegym.net/compendium-lvx.html#410